pkg/system/phase1_verifying.go

```diff
@@ -171,6 +180,18 @@ func (r *Reconciler) CheckSystemCR() error {
 			fmt.Sprintf("Autoscaler %s missing prometheusNamespace property ", r.NooBaa.Spec.Autoscaler.AutoscalerType))
 	}
 
+	// Rejecting NooBaa CR if 'BucketLoggingPVC' is not provided for 'guaranteed' logging and
```

Comment:
I think we can combine both check in 69 with this to one if that check both case. in anyway we want to check the bucketLoggingPVC just in case we use BucketLoggingTypeGuaranteed

Reply:
Make sense I will update it. Thanks.